base/PHP5Daemon.php

```diff
@@ -100,6 +106,7 @@ private function isPHPCGIProcess(int $pid): bool {
 
   protected function getEnvironmentVariables(): Map<string, string> {
     return Map {
+      'OSS_PERF_TARGET' => (string) $this->target,
```

Comment:
Yes, this is deliberate.  I mentioned it in the commit message.  Rather than running php-cgi or hhvm directly, I'm running a bash script which eventually execs the intended unit under test.  The bash script sets up some additional information used to differentiate different test runs to our profiling system.

Good point about making this work for the hhvm runner.  I hadn't actually gotten that part of the equation working.